packages/@aws-cdk/aws-config/test/integ.rule.lit.ts

```diff
@@ -38,5 +39,7 @@ class ConfigStack extends cdk.Stack {
   }
 }
 
-new ConfigStack(app, 'aws-cdk-config-rule-integ');
+new ConfigStack(app, 'aws-cdk-config-rule-integ', {
+  env: { account: '123456789012', region: 'us-east-1' },
```

Comment:
```suggestion
```

The integ tests stacks should be environment agnostic.

Reply:
Let me leave this comment as it is. It is certainly not a process that asks for region and account ID. However, if it is randomized, the hash seed of the snapshot test will change every time, and the test will not converge.

Reply:
I think it's probably because when the env is not defined then the account and region are tokens.

I left a review comment on what you need to updated.